cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -712,7 +721,26 @@ declare class Worker extends EventTarget {
        * hello world2
        *
        */
-      deno?: boolean;
+      deno?: {
```

Comment:
I wonder if we should keep `deno: true` for now so we don't break user code 🤔  it's unstable but it would be quite disruptive for a patch release

Reply:
Yeah, let's keep `true` for backwards compat. However I'll remove docs that use it and put a couple comments so we can safely remove it in 2.0

Reply:
Let's remove it in 1.7.0.